tests/testthat/test-tbl_survfit.R

```diff
@@ -1,21 +1,166 @@
-test_that("tbl_survfit() works", {
-  expect_error(
+test_that("tbl_survfit(time) works", {
+  expect_silent(
     trial |>
       tbl_survfit(
         include = trt,
         y = "Surv(ttdeath, death)",
         times = 12
-      ),
-    NA
+      )
+  )
+})
+
+test_that("tbl_survfit(probs) works", {
+  expect_silent(
+    trial |>
+      tbl_survfit(
+        include = trt,
+        y = "Surv(ttdeath, death)",
+        probs = 0.5
+      )
   )
+})
 
+test_that("Using both times and probs errors correctly", {
   expect_error(
     trial |>
       tbl_survfit(
         include = trt,
         y = "Surv(ttdeath, death)",
+        times = 12,
         probs = 0.5
       ),
-    NA
+    regexp = "Specify one and only one of arguments `times` and `probs`."
+  )
+})
+
+test_that("tbl_survfit(statistic) works", {
+  expect_silent(
+    tbl <- trial |>
+      tbl_survfit(
+        include = trt,
+        y = "Surv(ttdeath, death)",
+        times = 12,
+        statistic = "{estimate} [{conf.low} - {conf.high}]"
+      )
+  )
+  expect_equal(
+    tbl$table_body$stat_1,
+    c(NA, "91% [85% - 97%]", "86% [80% - 93%]")
+  )
+})
+
+test_that("tbl_survfit(label) works", {
+  # label currently WILL NOT OVERRIDE PREXISTING LABELS
+  remove_labels <- function(x) {
+    attr(x, "label") <- NULL
+    return(x)
+  }
+
+  expect_silent(
+    trial |>
+      dplyr::mutate(dplyr::across(everything(), remove_labels)) |>
+      tbl_survfit(
+        include = c(trt, grade),
+        y = "Surv(ttdeath, death)",
+        times = 12,
+        label = list(trt = "TREATMENT", grade = "GRADE")
```

Comment:
@ddsjoberg only thing I've "found" is that if there's pre-existing labels like there is in `trial` rn, it won't work, not sure if that was the intended consequence

Reply:
It'll overwrite if its just a single value though

Reply:
I think this should be fixed now, thank you!!!